File: Python/1160.py
t = int(input())

for i in range(0, t):
    entrada = input().split()
    popA, popB = int(entrada[0]), int(entrada[1])
    gA, gB = float(entrada[2]), float(entrada[3])

    anos = 0

    while popA <= popB:
        crescA = int(popA * (gA/100))
        crescB = int(popB * (gB/100))
        popA += crescA
        popB += crescB
        anos += 1
        if anos > 100:
            print("Mais de 1 seculo.")
            break
    
    if anos <= 100:
        print(f"{anos} anos.")

File: C/1160.c
#include <stdio.h>

int main(){

    int t, pa, pb, anos = 0;
    double g1, g2;

    scanf("%i", &t);

    for(int i = 0; i < t; i++){

        scanf("%i %i %lf %lf", &pa, &pb, &g1, &g2);

        while(pa <= pb){
            pa += pa * (g1/100);
            pb += pb * (g2/100);
            anos += 1;

            if(anos > 100){
                printf("Mais de 1 seculo.\n");
                break;
            }
        }

        if(anos <= 100){
            printf("%i anos.\n", anos);
        }
        

        anos = 0;
    }

    return 0;
}